src/main/java/uk/gov/hmcts/reform/sscs/helper/service/HearingsServiceHelper.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.sscs.helper.service;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static java.util.Objects.isNull;
+
+public final class HearingsServiceHelper {
+
+    private HearingsServiceHelper() {
+    }
+
+
+    public static void updateHearingId(HearingWrapper wrapper, HearingResponse response) {
+        getSchedulingAndListingFields(wrapper).setActiveHearingId(response.getHearingRequestId());
+    }
+
+    public static void updateVersionNumber(HearingWrapper wrapper, HearingResponse response) {
+        getSchedulingAndListingFields(wrapper).setActiveHearingVersionNumber(response.getVersionNumber());
+    }
+
+    public static SchedulingAndListingFields getSchedulingAndListingFields(HearingWrapper wrapper) {
+        SscsCaseData caseData = wrapper.getUpdatedCaseData();
```

Comment:
Have we unified original and updated case data? Do we still need the updated case data?